tap_hubspot/__init__.py

```diff
@@ -319,6 +394,14 @@ def gen_request(STATE, tap_stream_id, url, params, path, more_key, offset_keys,
         while True:
             data = request(url, params).json()
 
+            if v3_fields:
```

Comment:
In a few spots below here, we call `data[path]`. Should we check that this exists and if not, raise an exception? That would make this code slightly safer and provide a more obvious failure.